eduNEXT/tutor-contrib-edunext-distro
# fix: allow session cookie domain in multitenancy

This PR creates the SESSION_COOKIE_DOMAIN in the LMS for tenants and microsites when isn't a subdomain of lms.mango.edunext.link.

**Note**: The cookie is only created with the login, it's not created through the admin console.

**How to test**

1. Use this branch
2. Create a tenant o microsite, you can use the tenant config by default like a guide, just remember to change the url, for example,  `microsite-a.mango.edunext.link`
3 .Login to the site 
4. Open the browser devtools and check the cookies and check for `edx-jwt-cookie-header-payload`
